src/utils/validationUtil.ts

```diff
@@ -1,3 +1,50 @@
 import * as validation from './formErrorUtil';
 
-export const required = (value: string) => validation.isRequired(value) ? undefined : 'ERRORS.FIELD_REQUIRED';
+type IValidation = (value: string) => string | undefined | object;
+
+type IValidationWithParamNumber = (param: number) => (value: string) => string | undefined | object;
```

Comment:
IValidationWithParamNumber and IValidationWithParamNumber are almost the same. you can combine these types